### PR TITLE
Bug 2080679: fix oc related test to land k8s 1.24

### PR DESCRIPTION
**Pull request by soltysh**
This should solve:
- https://bugzilla.redhat.com/show_bug.cgi?id=2080679
- https://bugzilla.redhat.com/show_bug.cgi?id=2081018

### CODE REVIEW

File: test/extended/testdata/cmd/test/cmd/secrets.sh [hunk 1]
```diff
@@ -150,9 +150,3 @@ os::cmd::expect_success 'oc secret unlink --help'
 
 echo "secrets: ok"
 os::test::junit::declare_suite_end
-
```
Comment (soltysh) [inline test/extended/testdata/cmd/test/cmd/secrets.sh:153]: These are being deprecated in https://github.com/openshift/oc/pull/1126 and also don't make sense after we land k8s 1.24, so it's better to get rid of them entirely.

**Comment by stlaz** [comment]
/hold
/test e2e-agnostic-cmd
lets see at least some reasonable results from this test

**Pull request merged**